docs/Getting_Started.md

```diff
@@ -1,17 +1,57 @@
 # Getting Started
 
-There are many places to start start in ElectionGuard. If you're looking to develop see below, if you just want to research see the [guide](guide/Overview.md). 
+There are many ways to adopt or contribute to ElectionGuard. It is an open source platform and depends on an active community and ecosystem.
 
-## Latest
+The ElectionGuard SDK comprises three Github repos:
 
-These are the latest projects from ElectionGuard. If you're looking to contribute, these are the best places to start.
+* ElectionGuard Specification
+* ElectionGuard Python
+* ElectionGuard C++
 
-[`electionguard-python`](guide/Core_Component.md#python)
+Whenever possible and as a guiding principle, the evolution of the ElectionGuard SDK will be driven by the use cases and priorities of the community and in service of best-practice implementations of real-world, secret-ballot public elections
 
-[`electionguard-web-api`](guide/Implementations.md)
+!!! info "Ways to get involved and find out what's actively being worked on"
+    If you have questions or ideas, there are multiple ways to get feedback and help:
+    
+    * Post questions or ideas in our [***discussion boards***](https://github.com/microsoft/electionguard/discussions) 
+    * Join our [monthly ***office hours videoconference***](https://github.com/microsoft/electionguard/discussions/61)
+    * Check out and volunteer to work on issues in the repos with tags of [**good first issue**](https://github.com/microsoft/electionguard-python/labels/good%20first%20issue), [**help wanted**](https://github.com/microsoft/electionguard-cpp/labels/help%20wanted) and, as always, [**documentation**](https://github.com/microsoft/electionguard/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation)
 
-## Language
 
-Pick your language of preference:
+## ElectionGuard Specification
 
-[![](images/python-language.png)](guide/Core_Component.md#python)[![](images/c++-language.png)](guide/Core_Component.md#c++) [![](images/c-language.png)](guide/Core_Component.md#c_1) [![](images/c-sharp-language.png)](guide/Core_Component.md#c_2) [![](images/java.png)](https://github.com/JohnLCaron/electionguard-java)
+The ElectionGuard Specification repo addresses the mathematical and conceptual underpinnings of end-to-end verifiability in a manner that comfortably accommodates post-election audits.
+
+The specification houses the core data constructs, tests, and datasets that we recommend to ensure portability and data integrity.
+
+[:fontawesome-brands-github: **ElectionGuard Specification**](./spec/0.95.0/1_Overview.md)
+## ElectionGuard Python
+
+![Python](images/python-language.png)
+
+The ElectionGuard Python library is the *reference implementation* of the ElectionGuard SDK. It covers the entire suite of functionality and processes necessary to implement an end-to-end verifiable election as part a voting system: 
+
+* Key generation ceremony
```

Comment:
I feel like these should be capitalized
Also are we changing this to `Key Generation Ceremony` over `Key Ceremony`
If so `Tally Decryption Ceremony` ...